service/service_test.go

```diff
@@ -38,7 +38,16 @@ import (
 func TestServiceCreation(t *testing.T) {
 	r := metrics.NewTestStatsReporter()
 	r.CountersWG.Add(1)
-	scope, closer := tally.NewRootScope("", nil, r, 50*time.Millisecond, tally.DefaultSeparator)
+	opts := tally.ScopeOptions{
+		Tags:           nil,
```

Comment:
Well, I was afraid linter would complain, because, we can extend a struct and forget some mandatory fields. But it didn't. Let's get dangerous.